access-modifier-checker/src/main/java/org/kohsuke/accmod/impl/Checker.java

```diff
@@ -383,24 +383,38 @@ public String getProperty(String key) {
         };
     }
 
+    private static String topLevelClass(String a) {
+      int i = a.indexOf('$');
```

Comment:
By the way this is not technically correct since `$` is a legal identifier character, but in practice it is so rarely used that I think this is fine.